benchmark/README.md

```diff
@@ -43,6 +43,8 @@ To harness the plot chart feature, first ensure that matplotlib is installed (re
 bazel run -c opt --//:has_rtti --//:has_matplotlib //benchmark/path/to/target:target_name -- -n <test_set_size>
 ```
 
+Build error with error message `error: undefined reference to ...` may occur when building the benchmark. This is because bazel does not know updates to python environment. To fix this (temporarily), add an empty line to [this file](/third_party/py/python_configure.bzl) and rebuild the benchmark then remove the empty line so that python related builds are triggered.
```

Comment:
```suggestion
A build error with error message `error: undefined reference to ...` may occur when building the benchmark. This is because bazel does not know about updates to the python environment. To fix this (temporarily), add an empty line to [this file](/third_party/py/python_configure.bzl) and rebuild the benchmark then remove the empty line so that python related builds are triggered.